### PR TITLE
Enhancements/a11y fixes

### PR DESCRIPTION
**Pull request by baz1414**
Added a minor change to templates with the `svg` objects for the players to include a `title` so they can be read by screen readers. 

would you be so kind as to tag the pull request with `hacktoberfest - accepted` if you end up approving the pull?

### CODE REVIEW

File: api/templates/spotify.compact.html.j2 [hunk 1]
```diff
@@ -1,4 +1,5 @@
-<svg width="320" height="{{height}}" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<svg width="320" height="{{height}}" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" aria-labelledby="catTitle">
+  <title id="catTitle">Now playing on Spotify</title>
```
Comment (bluenex) [inline api/templates/spotify.compact.html.j2:2]: Why is it `catTitle`? This template renders a card, should it be `cardTitle` or something else instead?

**Review by bluenex — COMMENTED** [review]
Also, shall we have `role` too? Is it necessary or useful for accessibility tool as well?

I have found it on https://developer.paciellogroup.com/blog/2013/12/using-aria-enhance-svg-accessibility/

File: api/templates/spotify.compact.html.j2 (hunk 1 above)
Comment (baz1414) [reply api/templates/spotify.compact.html.j2:2]: Great catch! Yes, `cardTitle` would be much better and was the intended name. I'll get that change.

**Comment by baz1414** [comment]
> Also, shall we have `role` too? Is it necessary or useful for accessibility tool as well?
> 
> I have found it on https://developer.paciellogroup.com/blog/2013/12/using-aria-enhance-svg-accessibility/

Great suggestion.

**Pull request merged**